src/recipes.jl

```diff
@@ -174,6 +174,18 @@ end
 end
 @deps scatterpath path scatter
 
+# ---------------------------------------------------------------------------
+# pixel
+
+@recipe function f(::Type{Val{:pixel}}, x, y, z)
+    x := x
+    y := y
+    seriestype := :pixel
```

Comment:
```suggestion
    seriestype := :scatter
```

Reply:
Perhaps I do not understand the recipe system, but in a sense, pixel could be supported natively by both GR and Pyplot. However, others might not support it, in that case it should fallback to :scatter. Are we on the same page?

Reply:
Hello @isentropic, sorry I am not able to work today. I have high fever. As soon as I start feeling better I will quickly read the recipe resource shared by @BeastyBlacksmith, then I might be able to understand things better.

Reply:
@BeastyBlacksmith, since pyplot and gr supports the pixel plots natively, we can update the natively supported series to handle `:pixel`. For the other plots we can add pixel series to them and make them fallback to a scatter plot.

Reply:
Of course, feel free to take your time

Reply:
Backends declare in `backends.jl` which seriestypes they support natively and for those this recipe will not be called.

Reply:
Hello @isentropic, I am finally recovered from the fever and continuing my work on this PR. I will push some changes soon.

Reply:
So I read the resource shared by @BeastyBlacksmith. He was right about the `seriestype := :pixel` thing, that it should be changed to `seriestype := :scatter`. I guess, this is due to the recursive calling of different recipes.